pkg/terraform.go

```diff
@@ -114,6 +114,26 @@ func (e *Executor) generateTfVarsFile(creds TfBackend) error {
 	return nil
 }
 
+func (e *Executor) getModules() error {
+	// create a modules directory within the targetted tf repo directory
+	moduleDir := fmt.Sprintf("%s/%s/%s/remote-modules", e.workdir, e.TfRepoCfg.Name, e.TfRepoCfg.Path)
```

Comment:
Nice! This makes it clear although we will have to inform tenants in our docs to specify this path.